src/parser.ts

```diff
@@ -1334,13 +1334,14 @@ export class Parser {
         return expr;
     }
 
-    // ECMA-262 12.6 Multiplicative Operators
-    // ECMA-262 12.7 Additive Operators
-    // ECMA-262 12.8 Bitwise Shift Operators
-    // ECMA-262 12.9 Relational Operators
-    // ECMA-262 12.10 Equality Operators
-    // ECMA-262 12.11 Binary Bitwise Operators
-    // ECMA-262 12.12 Binary Logical Operators
+    // ECMA-262 12.6 Exponentiation Operators
```

Comment:
We need to seriously consider that for the later editions. It's still problematic either way for ES2017, the chapter index is still not known (for e.g. stage-4 async function) and the URL doesn't exist yet (http://ecma-international.org/ecma-262/8.0), until the spec is almost finalized.